build/RunHelixTests.yml

```diff
@@ -1,6 +1,8 @@
 name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
 jobs:
 - job: Build
+  condition: 
```

Comment:
Ignore changes in this file. This is just for testing. Did not mean to push it here.